test/functional/discovered_hosts_controller_test.rb

```diff
@@ -275,7 +275,8 @@ def test_multiple_reboot_failure
     ::ForemanDiscovery::NodeAPI::PowerService.any_instance.expects(:reboot).returns(false)
     post :submit_multiple_reboot, params: {:host_ids => host.id}, session: set_session_user(User.current)
     assert_redirected_to discovered_hosts_url
-    assert_equal "Errors during reboot: #{host.name}: failed to reboot", flash[:error]
+    require 'pry'; binding.pry;
```

Comment:
wat? :)

Reply:
sorry. forgot to remove it after testing.